app/_locales/en/messages.json

```diff
@@ -2648,6 +2648,12 @@
   "usePhishingDetectionDescription": {
     "message": "Display a warning for phishing domains targeting Ethereum users"
   },
+  "useStaticTokenList": {
+    "message": "Use Token Detection"
```

Comment:
Doesn't `useStaticTokenList` mean the opposite of `Use Token Detection`?

Reply:
The controller flag that will be updated with the toggle button is named `useStaticTokenList`, so I went with the same name for the key.

Reply:
Per our conversation on #11398 I would vote to change this to `useTokenDetection`!

Reply:
This will be updated after #11398 is merged